statistics/histogram.go

```diff
@@ -441,6 +457,10 @@ func (hg *Histogram) SplitRange(ranges []*ranger.NewRange) []*ranger.NewRange {
 			ranges[0].LowExclude = true
 		}
 	}
+	valid := validRange(split[len(split)-1])
```

Comment:
can other range rather than the last one be invalid?

Reply:
First, all of the `ranges` are valid, then for the split range, we can make sure that they are valid, so the only the last one could be invalid.